src/MCPServer/lib/server/shared_dirs.py

```diff
@@ -300,6 +309,7 @@ def create():
         "www/AIPsStore/transferBacklog/originals",
         "www/DIPsStore",
     )
+    dirs = dirs + generate_transfer_watched_dirs()
```

Comment:
At this point where we're manipulating the data I'd sugget to stop using tuples.